src/scripts/components/stories/story-gallery-item/story-gallery-item.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {FunctionComponent} from 'react';
+import cx from 'classnames';
+
+import styles from './story-gallery-item.module.styl';
+
+interface Props {
+  children: React.ReactNode[];
+  currentIndex: number;
+  showLightbox: boolean;
+}
+
+const StoryGalleryItem: FunctionComponent<Props> = ({
+  children,
+  currentIndex,
+  showLightbox
+}) => {
+  const containerWidth = children.length * 100;
+  const itemWidth = 100 / children.length;
```

Comment:
Could you add a comoment why you are multiplying / dividing by 100?

Reply:
I reused that code from image gallery, but yes, may be helpful here.